ping.h

```diff
@@ -28,7 +28,7 @@ struct ping_resp {
 };
 
 bool ping_start(struct ping_option *ping_opt);
-void ping(const char *name, int count, int interval, int size, int timeout);
-bool ping_start(IPAddress adr, int count, int interval, int size, int timeout);
+//void ping(const char *name, int count, int interval, int size, int timeout);
+bool ping_start(struct ping_option *ping_o,IPAddress adr, int count, int interval, int size, int timeout);
```

Comment:
this need to be refactored.

Reply:
according with the previous pull request.

Reply:
Then I propose that I will change to something like: 
- Make the ping_option pointer as "optional" , something like: struct ping_option *ping_o = NULL , and inside the function we will compute all the "Statistics' and we will call the callback function only if we have a valid ping_o pointer. This way the ping() function will call ping_start() without ping_o pointer and PingClass::ping() can call it WITH ping_o parameter

What do you think ?

Reply:
yes.I think this is the correct aproach here.